### PR TITLE
Added some some boilerplate for the views and components

### PR DESCRIPTION
**Pull request by JoelRice**

### CODE REVIEW

File: src/fetchRequests.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//This is not our current baseURL. It is just here as a place holder.*needs updated with new API
+export const baseURL = "https://socialapp-api.herokuapp.com/";
```
Comment (gmferise) [inline src/fetchRequests.js]: Will either be `localhost:3000` or `https://subdued-fog-mouth.glitch.me/`

File: src/fetchRequests.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  })
+    .then((res) => res.json())
+    .then((res) => {
+      return res;
+    });
+};
+//Logout Fetch Request *needs updated with new API
+export const logoutRequest = (token) => {
+  return fetch(baseURL + "auth/logout", {
+    headers: { Authorization: "Bearer " + token },
```
Comment (gmferise) [inline src/fetchRequests.js]: Instead of the Authorization header:
```
body: {
  token: token
}
```

File: src/store/store.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import create from "zustand";
+import { devtools, redux } from "zustand/middleware";
+
+// define the store's initial state
+const initialState = { user: { token: "" }, messages: [] };
```
Comment (gmferise) [inline src/store/store.js]:
```
const initialState = {
  token: '',
  toast: {
    text: '',
    color/statusCode/whatever: ''
  }
}
```

File: src/store/store.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+// set action types
+export const LOGIN = "LOGIN";
```
Comment (gmferise) [inline src/store/store.js]:
```
export const actions = {
  LOGIN: 'LOGIN',
  ...
}
```

File: src/views/adopt.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+
+function Adopt(props) {
+  return <div className="Adopt-page"></div>;
```
Comment (gmferise) [inline src/views/adopt.js]: Class names are usually all lowercase: `adopt-page`

File: README.md [hunk 6]
```diff
@@ -1 +1,4 @@
 # kapstone-frontend
+
+Authors:Gavin Ferise, Ur
```
Comment (gmferise) [inline README.md]: Maybe should write all our names :wink:

**Review by gmferise — CHANGES_REQUESTED** [review]

File: src/views/adopt.js (hunk 5 above)
Comment (gmferise) [reply src/views/adopt.js]: You'll have to update a lot, have fun

File: src/fetchRequests.js (hunk 2 above)
Comment (JoelRice) [reply src/fetchRequests.js]: I replaced headers: {} with body: {}

**Review by gmferise — APPROVED** [review]

**Pull request merged**